src/ServiceControl.Transports.SQS/ServiceControl.Transports.SQS.csproj

```diff
@@ -9,7 +9,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus.AmazonSQS" Version="4.4.0" />
+    <PackageReference Include="NServiceBus.AmazonSQS" Version="5.0.0-PullRequest0426.333" />
```

Comment:
@mauroservienti is that intentional to reference a pre-release?

Reply:
@ramonsmits this is TF work that @mauroservienti and @yvesgoeleven are working on